ui/apps/dashboard/src/app/(dashboard)/settings/integrations/vercel/useUpdateVercelIntegration.ts

```diff
@@ -68,7 +68,7 @@ export default function useUpdateVercelIntegration(initialVercelIntegration: Ver
     const createVercelAppPromises = projectsToCreate.map((project) =>
       createVercelApp({
         input: {
-          path: project.servePath ?? null,
```

Comment:
Ah, so is this that `servePath` was nullish because we wanted the default, but the GraphQL endpoint is then using that as a value as it has been defined?

Would it be reasonable to ensure the GraphQL API handles this instance better?

Reply:
Like make the Go code in GraphQL handle defaults instead of the schema?